src/patternfly/components/ClipboardCopy/clipboard-copy-group-toggle.hbs

```diff
@@ -1,4 +1,4 @@
-<button class="pf-c-clipboard-copy__group-toggle{{#if clipboard-copy-group-toggle--modifier}} {{clipboard-copy-group-toggle--modifier}}{{/if}}"
+<div class="pf-c-clipboard-copy__group-toggle{{#if clipboard-copy-group-toggle--modifier}} {{clipboard-copy-group-toggle--modifier}}{{/if}}"
```

Comment:
@mcoker @mattnolting I removed the hbs file for `pf-c-clipboard-copy__group-copy` because it was only just a wrapper. I didn't remove the hbs for `pf-c-clipboard-copy__group-toggle` because its used to determine whether its expanded or not, what do you both think about that?